### PR TITLE
Feat/enhance chart component 72

### LINKED ISSUE #72: Enhance Line Chart Component to Display Up to Three Lines
## Enhance Line Chart Component for Multiple (Up to Three) Lines Selection

Image

**Description:**
Enhance the existing line chart component in our UI kit to allow for displaying up to three lines simultaneously. Users should be able to select which lines they wish to view.

**Acceptance Criteria:**
- The line chart component can display 1 to 3 lines based on user selection.
- Users have a UI element (e.g., checkboxes) to select which lines to display.
- Ensure the component remains responsive and visually coherent with multiple lines.

### LINKED ISSUE #76: Graph Numbers are Cut Off
The numbers on the Y axis of the graph are cut off.   Desktop , Brave browser

Image

### PR DESCRIPTION
**Pull request by Torresmorah**
### What does this PR do?

- Resolve #72 
- Resolve #76 
- Allow to have as many lines and bottom items as needed
- Allow to enable and disable the lines in the chart
- Allow to pass a function to format the numbers
- Add a CardComponent to replace FlatComponent in the Dashboard
- Remove currency label from the cards

### Steps to test

1. Run the project locally
2. Go to Dashboard component section
3. Check the Component

### Screenshots

**Desktop**

imagen
imagen

**Tablet**

imagen

**Mobile**

imagen

### Label Issue

**Before**
imagen
**After**
imagen

### CODE REVIEW

**Comment by xavier506** [comment]
@Torresmorah  can you please also take a look at this issue #76

**Review by Edwinjrc — APPROVED** [review]
LGTM

**Comment by leisterfrancisco** [comment]
@Edwinjrc @Torresmorah dollar symbol in the top left section is repetitive, `$...usdc`

File: src/components/Graphic/index.tsx [hunk 1]
```diff
@@ -1,53 +1,78 @@
...
 } from "recharts";
+import clsx from "clsx";
 import {
   NameType,
   ValueType,
 } from "recharts/types/component/DefaultTooltipContent";
 
 import { Container } from "../Container";
 import { FluctuationComponent } from "../FluctuationComponent";
+import { Point } from "../Point";
```
Comment (leisterfrancisco) [inline src/components/Graphic/index.tsx:20]: let's leave a blank line to divide project imports and local imports

File: src/stories/Dashboard.stories.tsx [hunk 2]
```diff
@@ -1,97 +1,104 @@
...
+]
+
+const formatValue = (value: number) => {
+  const newValue = value.toFixed(2)
+
+  const [integer, decimal] = newValue.toString().split('.')
+
+  return `$${integer.replace(/\B(?=(\d{3})+(?!\d))/g, ',')}${
+    decimal ? '.' + decimal : ''
+  }`
```
Comment (leisterfrancisco) [inline src/stories/Dashboard.stories.tsx:77]: There is already a method for this type of formatting, please use it

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString#using_tolocalestring

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: src/components/Dashboard/index.tsx [hunk 3]
```diff
@@ -3,60 +3,60 @@ import clsx from "clsx";
...
+import { GraphicDataType, Graphic, SeriesDataType } from "../Graphic";
 import lendIcon from "../../../public/assets/lending.svg";
 import dolarIcon from "../../../public/assets/dolar-icon.svg";
 import { FluctuationComponent } from "../FluctuationComponent";
 
+export type BottomContainerItem = {
+  label: string;
+  value: number;
+  fluctuation: string;
+};
```
Comment (leisterfrancisco) [inline src/components/Dashboard/index.tsx:16]: is `;` required?

Comment (leisterfrancisco) [reply src/components/Dashboard/index.tsx:16]: maybe prettier is not configured correctly, would be nice to set it up in that case

**Review by leisterfrancisco — APPROVED** [review]

**Review by xavier506 — APPROVED** [review]
lgtm

**Pull request merged**